program/rust/build.rs

```diff
@@ -9,6 +9,7 @@ fn main() {
 
     //generate and write bindings
     let bindings = Builder::default()
+        .clang_arg("-I../../../solana/sdk/bpf/c/inc/")
```

Comment:
required to make bindgen find the solana header files.

hardcoding this path is obviously bizarre. however, the C makefile already does this, so I'm not making the situation any worse.